### PR TITLE
Improve non-verbose mode

### PR DESCRIPTION
**Pull request by tuler**
Add the application output to non-verbose mode.

### CODE REVIEW

**Comment by tuler** [comment]
@endersonmaia I changed the base branch, so it's ready for review (not yet to main)

**Review by endersonmaia — CHANGES_REQUESTED** [review]
The first commit is already on the target branch.

File: apps/cli/src/commands/run.ts [hunk 1]
```diff
@@ -71,28 +76,19 @@ export default class Run extends Command {
...
         ];
 
-        const mode = flags.verbose ? [] : ["--detach", "--wait"];
-
-        this.log("Starting node...");
         try {
             // run compose environment
-            await execa("docker", [...args, "up", ...mode], {
-                stdio: "inherit",
+            const up = execa("docker", [...args, "up"], {
```
Comment (endersonmaia) [inline apps/cli/src/commands/run.ts:81]: I tested this, and it kinda worked, but the message `?Press enter... ` doesn't show up. 

```suggestion
            const up = execa("docker", [...args, "up", "--attach", "validator"], {
```


```
+] Building 0.0s (0/0)                                                                                                                                                                                                                                                     
[+] Running 6/4
 ✔ Network 88dcfaab_default            Created                                                                                                                                                                                                                         0.1s 
 ✔ Container 88dcfaab-redis-1          Created                                                                                                                                                                                                                         0.0s 
 ✔ Container 88dcfaab-anvil-1          Created                                                                                                                                                                                                                         0.0s 
 ✔ Container 88dcfaab-database-1       Created                                                                                                                                                                                                                         0.0s 
 ✔ Container 88dcfaab-dapp_deployer-1  Created                                                                                                                                                                                                                         0.0s 
 ✔ Container 88dcfaab-validator-1      Created                                                                                                                                                                                                                         0.0s 
Attaching to 88dcfaab-validator-1
88dcfaab-validator-1  | GraphQL server listening at http://localhost:8080/graphql
88dcfaab-validator-1  | Inspect server listening at http://localhost:8080/inspect
88dcfaab-validator-1  | 2023-06-06 21:01:47.141826 info remote-machine pid:232 ppid:75 Initializing server on localhost:0
88dcfaab-validator-1  | time="2023-06-06T21:01:47Z" level=info msg="Configuration loaded from file: /etc/traefik/traefik.yaml"

```

Comment (tuler) [reply apps/cli/src/commands/run.ts]: Yeah, this was not nice. I remove the message and the wait.
And then I'll investigate further the SIGTERM situation.

Comment (tuler) [reply apps/cli/src/commands/run.ts]: Don't need the `"--attach", "validator"`.
I fixed an issue, it had a detached true, before.

Comment (endersonmaia) [reply apps/cli/src/commands/run.ts]: But with `--attach validator` you get logs only for the validator, and hides logs for the others, and have CTRL + C working.

Comment (endersonmaia) [reply apps/cli/src/commands/run.ts]: I'm wrong about CTRL + C.

File: packages/rollups-node/s6-overlay/s6-rc.d/stream-cleanup/script/up.sh [hunk 2]
```diff
@@ -1,9 +1,8 @@
...
-if [ -z ${ENABLE_STREAM_CLEANUP+x} ];then
-    printf "stream-cleanup: ENABLE_STREAM_CLEANUP not set, skipping cleanup\n"
-else
+if [ ! -z ${ENABLE_STREAM_CLEANUP+x} ];then
     printf "stream-cleanup: ENABLE_STREAM_CLEANUP is set, cleaning redis streams...\n"
-    redis-cli -h ${REDIS_ENDPOINT} DEL chain-${CHAIN_ID}:dapp-${DAPP_CONTRACT_ADDRESS:2}:rollups-input;
     redis-cli -h ${REDIS_ENDPOINT} DEL chain-${CHAIN_ID}:dapp-${DAPP_CONTRACT_ADDRESS:2}:rollups-claims;
-fi
\ No newline at end of file
+    redis-cli -h ${REDIS_ENDPOINT} DEL chain-${CHAIN_ID}:dapp-${DAPP_CONTRACT_ADDRESS:2}:rollups-input;
```
Comment (endersonmaia) [inline packages/rollups-node/s6-overlay/s6-rc.d/stream-cleanup/script/up.sh:6]: I know you didn't added this, but the stream name is wrong.

```suggestion
    redis-cli -h ${REDIS_ENDPOINT} DEL chain-${CHAIN_ID}:dapp-${DAPP_CONTRACT_ADDRESS:2}:rollups-inputs;
```

**Review by endersonmaia — CHANGES_REQUESTED** [review]

File: apps/cli/src/commands/run.ts (hunk 1 above)
Comment (tuler) [reply apps/cli/src/commands/run.ts]: Hmm, that's true (about the log). Do you thing that's better?

Comment (tuler) [reply apps/cli/src/commands/run.ts]: And do you think verbose mode should attach to all?

Comment (endersonmaia) [reply apps/cli/src/commands/run.ts]: Yes.

You can still call `docker logs -f` for other containers if needed.

**Review by endersonmaia — APPROVED** [review]

**Pull request merged**